.gitignore

```diff
@@ -150,4 +150,7 @@ fabric.properties
 .idea/httpRequests
 
 # Android studio 3.1+ serialized cache file
-.idea/caches/build_file_checksums.ser
\ No newline at end of file
+.idea/caches/build_file_checksums.ser
+
+# Dependencies
+deps/*
```

Comment:
@chrisdiana Why do we need this?

Reply:
@ashtul when you initially are setting up tests on a new repo and run `make -C test/docker test PYTHON_VER=3` generated a `./deps` directory that was dumped into my root path with "readies" inside it. It seemed like someething that wasn't cleaned up that maybe should have so I added to `.gitignore` in case the cleanup failed on others environments. I can remove it if you like though.